src/account_response.ts

```diff
@@ -0,0 +1,73 @@
+import { Account as BaseAccount } from 'stellar-base';
+import forIn from 'lodash/forIn';
```

Comment:
Looks like this got rmed / added instead of moved, can you try to change this to a move so we can preserve change history?